Game2048/Game/Pages/Game2048Page.razor.css

```diff
@@ -0,0 +1,1555 @@
+@charset "UTF-8";
```

Comment:
Good idea on getting styles from the game page, however, are all these styles needed? and if so, this doesn't look like practicing baby steps 😨